packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -757,6 +757,10 @@ mixin WidgetsBinding on BindingBase, SchedulerBinding, GestureBinding, RendererB
 ///
 /// Initializes the binding using [WidgetsFlutterBinding] if necessary.
 ///
+/// Provide [customShaderWarmUp] if some specific Skia shaders need to be
```

Comment:
I think this can be elaborated a bit to make it more useful. Something like, "if the application has scenes that require the compilation of complex shaders, it may cause jank in the middle of an animation or interaction. Painting that scene here tells Flutter to precompute and cache the results of expensive operations so they can be reused later. This should only happen during the very first run of the Flutter application and is typically very small. It is advised that you add tracing here so that such pauses are easier to identify on the timeline. The applications main thread will not be blocked when Flutter is doing this work so you should no expect ANR warnings".

Something like that.

Reply:
Thanks! I've combined yours with my original documentation. I omit the tracing part because it's already added to `_warmUpSkiaShaderCompilations` which calls either [defaultShaderWarmUp] or [customShaderWarmUp].

CC @sfshaza2 for more API doc review.